pkgs/applications/graphics/pinta/default.nix

```diff
@@ -37,35 +40,41 @@ buildDotnetModule rec {
     owner = "PintaProject";
     repo = "Pinta";
     rev = version;
-    sha256 = "sha256-iOKJPB2bI/GjeDxzG7r6ew7SGIzgrJTcRXhEYzOpC9k=";
+    sha256 = "sha256-Bvzs1beq7I1+10w9pmMePqGCz2TPDp5UK5Wa9hbKERU=";
   };
 
-  postFixup = ''
-    # Rename the binary
-    mv $out/bin/Pinta $out/bin/pinta
+  # https://github.com/NixOS/nixpkgs/issues/38991
+  # bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
+  LOCALE_ARCHIVE = "${glibcLocales}/lib/locale/locale-archive";
 
-    # Copy desktop icons
-    for size in 16x16 22x22 24x24 32x32 96x96 scalable; do
-      mkdir -p $out/share/icons/hicolor/$size/apps
-      cp xdg/$size/* $out/share/icons/hicolor/$size/apps/
-    done
+  # Do the autoreconf/Makefile job manually
+  # TODO: use upstream build system
+  postBuild = ''
+    # Substitute translation placeholders
+    intltool-merge -x po/ xdg/pinta.appdata.xml.in xdg/pinta.appdata.xml
+    intltool-merge -d po/ xdg/pinta.desktop.in xdg/pinta.desktop
 
-    # Copy runtime icons
-    cp -r Pinta.Resources/icons/hicolor/16x16/* $out/share/icons/hicolor/16x16/
+    # Build translations
+    dotnet msbuild Pinta \
+      -p:ContinuousIntegrationBuild=true \
+      -p:Deterministic=true \
```

Comment:
BTW, would be nice to have those flags inside the `buildDotnetModule` environment (maybe something like `defaultDotnetFlags` :thinking: ?), since it is used everywhere inside it and we could use those same flags when needing to call `dotnet` commands manually.

Reply:
That's a good idea, thanks!